.github/workflows/prs.yml

```diff
@@ -1,15 +1,12 @@
 name: CI
-
-on: [push, pull_request]
-
+on:
+  pull_request:
```

Comment:
Are we sure that we do not want to execute these jobs again when they reach master?
```yml
on:
  push:
    branches: [master]
  pull_request:
```

Reply:
No, master gets a different flow that tests and does an automated release.